README.md

```diff
@@ -1,86 +1,130 @@
-# Example Client
-
-### Note:
-Enzyme is not ready for react 17 yet, so using an unofficial version and installing
-packages with `npm i --legacy-peer-deps` if using node v7.
-#### Goal:
-* Change to official version of enzyme when correct version is available.
-* Issue: https://github.com/enzymejs/enzyme/issues/2429
-
-## Project Library and Design Choices
-Below is a list of important libraries and dependecies for the project and what their purpose is.
+# DSNP Example Client
+
+This is an example client to help developers learn how to work with
+the [Distributed Social Networking Protocol
+(DSNP)](https://spec.dsnp.org/) and its
+[SDK](https://www.dsnp.org/sdk).  This client's purpose is to be
+simple and comprehensible for developers, not to have all the
+functionality and user-interface polish of an application intended for
+widespread general usage.
+
+The client talks to a local server (the "static server") that provides
+its own localized and isolated blockchain.  Please note that the
```

Comment:
The static server does not actually provide the blockchain; it's a local store for batch files and activity content that example-client can post to and retrieve from.  It doesn't even talk to the blockchain.  But this brings up the point that somewhere, probably a separate page, we should actually talk about where things are and what they're for.